android/orblibrary/src/main/java/org/orbtv/orblibrary/BrowserView.java

```diff
@@ -230,6 +226,26 @@ public void close() {
         destroy();
     }
 
+    public int getRenderingResolution() {
+        return mRenderingResolution;
+    }
+
+    private int getRenderingResolution(int rendering, int[] graphicsConfigIds) {
```

Comment:
This method probably should requires a version flag.
Thus,
For 203, it always returns rendering resolution, which is set in OrbSession.configuration
For 204, it returns a value, which is supported by graphics of App as well.